#Set ROS publishing rate - independet from the actual frame rate of the camera (it adjust dynamically)
frame_rate: 10.0

#Activate Downsampling and set the size of the generated voxels in meter. (0.05 = 5cm)
#This is helpful to reduce the bandwith in the network or to limit the data comming to your algorithm afterwards
downsampling: false
downsampling_voxel_size: 0.05

#Enable or Disable the projected range or/and intensity image
pointcloud: true
range_image: true
intensity_image: true

#Settings for the Auto Exposure
ampl_single: 1000.0
max_exposure: 32766 #multiplied by 0.00001241 is time in seconds
hdr_factor : 0.05
exposure_mode: 1 #-1: manual (set max_exposure), 0: default, 1: Auto Exposure, 2: Multi Exposure

#Pointcloud scaling
XYZ_to_m: 0.00037
